### DISCUSSION TITLE
How to track local changes

### DISCUSSION THREAD

**Discussion by emailvenkatm** [appwrite/appwrite#3629] [Q&A]
Hello, 

I am new to appwrite and I am really impressed with all the great stuff. One thing not clear to me is, how do you deploy the changes that you have done in your local? Like if you create a collection in your local how do you track those changes promote them to your environments?

**Comment by stnguyen90** [comment]
During development, the best way is to use the [Appwrite CLI](https://appwrite.io/docs/command-line) to create an [appwrite.json](https://appwrite.io/docs/command-line#appwriteJSON) file that manages your Functions and Collections. You can then point your CLI to [different instance](https://appwrite.io/docs/command-line#configuration) and then [deploy](https://appwrite.io/docs/command-line#deployFunctions). After production launch, I would not use the Appwrite CLI to deploy changes to Collections because doing so wipes the existing Collection to deploy the updated Collection.

**Reply by stnguyen90** [reply]
> Thanks for the response. So after the launch how do you deploy your changes then?

@emailvenkatm, for now, you can manually make the change in production or write scripts (using the Appwrite CLI or server SDK of your choice) to perform the change.

**Comment by emailvenkatm** [comment]
Thanks for the response. So after the launch how do you deploy your changes then? @stnguyen90
